content/v2.md

```diff
@@ -193,6 +193,37 @@ were created in and notices based on these objects will be delivered with
 the proper locale. i.e. if an issue is created with the "es" locale, all
 email to the creator will be delivered with Spanish translations.
 
+## Terms of Use
+
+### API Rate Limits
+
+The rate limit allows for up to 20 requests per minute. 
+
+Requests are associated with the originating IP address.
```

Comment:
I would drop this as creative people will start using multiple IP addresses.  If someone gets rate limited we can have a discussion with them about how it works.